cmd/admin-handlers-idp-config.go

```diff
@@ -354,6 +354,11 @@ func (a adminAPIHandlers) DeleteIdentityProviderCfg(w http.ResponseWriter, r *ht
 	var subSys string
 	switch idpCfgType {
 	case madmin.OpenidIDPCfg:
+		// idpCfgType is OpenidIDPCfg,Can't delete _ (Default)
+		if cfgName == madmin.Default {
+			writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(ErrAdminInvalidArgument), r.URL)
+			return
+		}
```

Comment:
this is an incorrect interpretation, default can be configured with the relevant target. A delete of default simply means "reset" it to default values. 

Each sub-system always has as `default` target of `_` that can never be "deleted" - there is nothing wrong here.

Reply:
Console UI should say "resetting" Default target to its default values.

Reply:
@harshavardhana No, I deleted _ and restarted minio, as shown in minio's log